src/services/pokemons.service.js

```diff
@@ -0,0 +1,13 @@
+import * as httpService from './httpBaseService';
+
+export const getAllPokemons = () => {
+    return httpService.get('pokemon?limit=200');
+}
+
+export const getPokemonDetails = (id) => {
+    return httpService.get(`pokemon/${id}`)
+}
+
+export const getPokemonCharacteristics= (id) => {
```

Comment:
I think fetching "Characteristics" is not needed.

Reply:
Wanted to render the pokemon description, but it's not in the same service for getting a pokemon's detail so I fetched the characteristics